src/utils/additionalProps.ts

```diff
@@ -6,6 +6,17 @@ import type {ETenantType} from '../types/api/tenant';
 
 import monitoringIcon from '../assets/icons/monitoring.svg';
 
+export function getMonitoringLink(
+    additionalProps?: AdditionalTenantsProps,
+    name?: string,
+    type?: ETenantType,
+): string | null {
+    if (!additionalProps?.getMonitoringLink) {
```

Comment:
what do u think about get rid of additionalProps here and use uiFactory?

Reply:
resolved in PM